src/canvas/quadFeature.js

```diff
@@ -143,5 +143,5 @@ var canvas_quadFeature = function (arg) {
 inherit(canvas_quadFeature, quadFeature);
 
 // Now register it
-registerFeature('canvas', 'quad', canvas_quadFeature);
+registerFeature('canvas', 'quad', canvas_quadFeature, {clr: false, img: true, 'img-full': false});
```

Comment:
Sounds reasonable.  The base class can document what the keys mean, too.  I'll make those changes.